### ISSUE TITLE
Is React 0.1.4 mandatory?

### ISSUE THREAD

**Issue by grigio** [thereactivestack/meteor-react-router-ssr#1]
```
meteor add reactrouter:react-router-ssr                                                                               ⏎
 => Errors while adding packages:             
                                              
While selecting package versions:
error: Conflict: Constraint react@0.1.4 is not satisfied by react 0.1.2.
Constraints on package "react":
* react@=0.1.2 <- top level
* react@0.1.4 <- reactrouter:react-router-ssr 0.1.2
* react@0.1.4 <- reactrouter:react-router 0.1.2 <- reactrouter:react-router-ssr 0.1.2
```

**Comment by eXon** [comment]
Update your Meteor's React package to 0.1.4 (NOT to confuse with React 0.14)
https://atmospherejs.com/meteor/react

I know this is confusing :)

**Comment by eXon** [comment]
Just a quick reminder: if you're using it with `ReactMeteorData`, don't forget to use the fixed version.

Let me know how this is working for you and if you need help for anything. This is brand new :)

**Comment by eXon** [comment]
@grigio actually, the version with ReactMeteorData fixed will be required to work fine (I guess 0.1.5 will).

**Comment by grigio** [comment]
I had an old local react package which did not update with `git pull`, thanks
